react/FileImageLoader/index.jsx

```diff
@@ -155,10 +157,10 @@ export class FileImageLoader extends Component {
 
   render() {
     const { src } = this.state
-    const { url } = this.props
+
     const { render, renderFallback } = this.props
-    if (url) {
-      return render(url)
+    if (this.context && this.context.url) {
+      return render(this.context.url)
```

Comment:
plutôt quelque chose comme ça : 

```
  render() {
    const { src } = this.state
    const { url } = this.context

    const { render, renderFallback } = this.props
    if (url) {
      return render(url)
```

dans quel cas this.context peut être undefined ? 🤔 uniquement si jamais on oublie le provider non ? Est-ce que cette condition est vraiment nécessaire ?

Reply:
J'ai eu un cas d'erreur avec le context undefined, donc je suppose que c'est nécessaire

Reply:
tu ne sais plus quel cas c'était ?